### ISSUE TITLE
being high on lithium while a mech causes weird shit

### ISSUE THREAD

**Issue by Serxule** [tgstation/tgstation#57757]
## Reproduction: be high on lithium while in mech, wait to twitch in the mech, and watch as confusion unfolds.

https://www.youtube.com/watch?v=ekziUXbO55w&ab_channel=Serxule 

demonstrated here

**Comment by SuperNovaa41** [comment]
Can you describe it? it's hard to see what exactly is happening

**Comment by ArcaneDefence** [comment]
Their camera focus gets disjointed from their mob.

**Comment by Serxule** [comment]
> Can you describe it? it's hard to see what exactly is happening

yeah what arcane said, camera stuck on mech despite lithium forcing me out

**Comment by TiviPlus** [comment]
Cause is the step() in lithium on_mob_life not checking that the users loc is a turf before moving them

**Comment by Pugie** [comment]
Don't drug and drive.
